packages/web/src/pages/edit-page/EditTrackPage.tsx

```diff
@@ -117,9 +118,11 @@ export const EditTrackPage = (props: EditPageProps) => {
       {trackStatus !== Status.SUCCESS || !coverArtUrl ? (
         <LoadingSpinnerFullPage />
       ) : (
-        <EditFormScrollContext.Provider value={scrollToTop}>
-          <EditTrackForm initialValues={initialValues} onSubmit={onSubmit} />
-        </EditFormScrollContext.Provider>
+        <UploadPreviewContextProvider>
```

Comment:
is it safe to also remove this provider from its previous location? or only once we've adopted the new flag?

If so, we can feature flag each of them now

Reply:
Need in both locations to work for upload and for edit, but might be able to move it to the form instead of the page to work for both